docs/index.md

```diff
@@ -45,72 +43,122 @@ Download the latest version of the CLI for your operating system and make sure t
+
+| Field      | Description                                                   | Format                                       |
+| ---------- | ------------------------------------------------------------- | -------------------------------------------- |
+| identifier | Version number or another unique identifier of the deployment | String                                       |
+| timestamp  | Time when the deployment finished                             | Number<br/>(Unix epoch timestamp in seconds) |
+|            | Commit identifiers included in the deployment                 | String<br/>(space-separated list)            |
+
+Using the Pulse CLI:
 
 ```sh
-./event-cli push change \
+./event-cli push deployment \
     --api-key "<API KEY>" \
-    --identifier "<commit identifier>" \
-    --timestamp "<Unix epoch timestamp in seconds>"
+    --identifier "<deployment identifier>" \
+    --timestamp "<timestamp>" \
+    <space-separated list of commit identifiers>
+```
+
+Using the webhook:
+
+```sh
+curl -X POST https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>
+    -d "api_key=<API KEY>"
+    -d "identifier=<deployment identifier>"
+    -d "timestamp=<timestamp>"
+    -d "<space-separated list of commit identifiers>"
 ```
 
-#### Incidents
+### Incidents
+
+Report an event to Pulse whenever there is a software release or infrastructure configuration change to production that results in degraded service and subsequently required remediation:
+
+-   The incident is **created** when you detect a service impairment or service outage in production.
+-   The incident is **resolved** when you apply a hotfix or patch, or when you rollback the changes to restore the service in production.
+
+Typically, it's possible to keep track of this information using your monitoring infrastructure or your ticketing system.
+
+Pulse uses these reports to calculate the metrics [Median time to recovery](metrics.md#median-time-to-recover) and [Change failure rate](metrics.md#change-failure-rate).
 
-Send information to Pulse whenever there is a change to production or a release to users that resulted in degraded service (e.g., service impairment or service outage) and subsequently required remediation (e.g., hotfix, rollback, fix forward, patch).
+You must send the following information when reporting incidents to Pulse:
 
 | Field             | Description                                    | Format                                       |
 | ----------------- | ---------------------------------------------- | -------------------------------------------- |
 | identifier        | A unique identifier of the incident            | String                                       |
 | timestampCreated  | Time when the incident started or was detected | Number<br/>(Unix epoch timestamp in seconds) |
 | timestampResolved | Time when the incident was resolved            | Number<br/>(Unix epoch timestamp in seconds) |
```

Comment:
Would it be easier for users to set up the integration if we split this event in two (one for the **creation** of the incident, and another for the **resolution**)?

I'm thinking about this since the beginning of the incident can be reported for example by a monitoring tool, and the resolution of the incident may be reported by a ticketing system or by a new deployment. :thinking:

Reply:
This is too advanced for now. I would say that usually all systems report the created time in the resolution which means we only need the resolution to be pushed.